### PR TITLE
NAS-121205 / 23.10 / Connected manual disk selection modal to pool wizard

### PR DESCRIPTION
**Pull request by RehanY147**

### CODE REVIEW

File: src/app/helptext/storage/volumes/manager/manager.ts [hunk 1]
```diff
@@ -41,6 +41,10 @@ export default {
   the encryption at the time you create the child dataset. For more information on encryption please\
   refer to the <a href="http://TrueNAS.com/docs/" target="_blank">TrueNAS Documentation hub</a>.'),
 
+  manual_disk_selection_message: T('VDEVs have been created through manual disk selection. To view or\
+  edit your selections, press the "Edit Manula Disk Selection" button below. To start again with the\
```
Comment (denysbutenko) [inline src/app/helptext/storage/volumes/manager/manager.ts:45]: Typo in "Manual"

**Review by denysbutenko — CHANGES_REQUESTED** [review]
Getting error in console
```
TypeError: Cannot read properties of undefined (reading 'rawSize')
    at rn (create-data-wizard-step.component.html:109:71)
```


https://user-images.githubusercontent.com/351613/230841791-6a3016da-a9b4-4466-89a9-af58fb6d6c89.mp4

**Review by undsoft — CHANGES_REQUESTED** [review]
1. If I save custom selection, Reset and then click Manual Disk selection, I expect to see the same autogenerated set of vdevs because values in dropdown were not changed.
2. If I open manual selection dialog and remove all vdevs and save, I don't expect it to say that custom layout was applied. Also if I press Edit again, I get a broken modal.
3. If I open manual selection, and make no changes, it would be good to also not say that custom layout was applied.

**Review by denysbutenko — APPROVED** [review]
Minor layout issue with the `Reset` button.
image

Otherwise, it looks great and works great for me.

**Review by undsoft — APPROVED** [review]
Looks good, thanks.

**Pull request merged**